libs/loopsidesheet/src/lib/ui-sidesheet/contentTable.tsx

```diff
@@ -0,0 +1,109 @@
+import { Loop } from '@cc-components/loopshared';
+import {
+  hasProperty,
+  LinkCell,
+  proCoSysUrls,
+  StatusCell,
+  statusColorMap,
+  TabTable,
+} from '@cc-components/shared';
+import { ColDef, ICellRendererProps } from '@equinor/workspace-ag-grid';
+import { LoopContent } from '../types';
+import { useGetLoopContent } from '../utils-sidesheet/useGetLoopContent';
+
+const columns: ColDef<LoopContent>[] = [
+  {
+    field: 'Tag',
+    valueGetter: (pkg) => pkg.data?.contentTagNo,
+    valueFormatter: (pkg) => {
+      if (pkg.data?.contentTagId) {
+        return proCoSysUrls.getTagUrl(pkg.data.contentTagId);
+      }
+      return '';
+    },
+    cellRenderer: (props: ICellRendererProps<LoopContent, string | null>) => {
+      if (props.valueFormatted) {
+        return <LinkCell url={props.valueFormatted} urlText={props.value ?? ''} />;
+      }
+      return null;
+    },
+  },
+  {
+    field: 'Description',
+    valueGetter: (pkg) => pkg.data?.description,
```

Comment:
Usually description fields are long texts and doesn't fit well in a table cell. Use the `DescriptionCell` in a `cellRenderer` to improve the display of it